app/services/session_encryptor.rb

```diff
@@ -1,4 +1,6 @@
 class SessionEncryptor
+  MARSHAL_SIGNATURE ||= 'BAh'.freeze
```

Comment:
should we put a little comment directly in this file so that the context of this isn't lost in the future?